modules/nuxt3-basic-auth.yml

```diff
@@ -0,0 +1,16 @@
+name: nuxt-basic-auth
+description: Nuxt 3 Module for Basic Authentication
+repo: kgierke/nuxt-basic-auth
+npm: '@kgierke/nuxt-basic-auth'
```

Comment:
I think we should just have one entry here, rather than two `yml` files pointing to the same repo/package